server/plugin_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin/plugintest"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetJob(t *testing.T) {
+	testServer := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
+		res.WriteHeader(http.StatusOK)
+	}))
+	assert.NotNil(t, testServer)
+	defer func() { testServer.Close() }()
```

Comment:
Nit: This can just be 
```suggestion
	defer testServer.Close()
```